Windows/Python/Install.ps1

```diff
@@ -45,8 +44,16 @@ $anacondaPath1 = "$env:USERPROFILE\Anaconda3"
 $anacondaPath2 = "C:\ProgramData\Anaconda3"
 
 if ((Test-Path $minicondaPath1) -or (Test-Path $minicondaPath2) -or (Test-Path $anacondaPath1) -or (Test-Path $anacondaPath2)) {
-    Write-Output "Miniconda or Anaconda is already installed. Skipping Miniconda installation."
-    Write-Output "If you wish to install Miniconda using this script, please uninstall the existing Anaconda/Miniconda installation and run the script again."
+    Write-Host "Miniconda or Anaconda is already installed. Skipping Miniconda installation."
+    Write-Host "If you wish to install Miniconda using this script, please uninstall the existing Anaconda/Miniconda installation and run the script again."
+    Write-Host " "
+    Write-Host "Please visit the following for more information: "
+    Write-Host " "
+    Write-Host "https://pythonsupport.dtu.dk/uninstall/conda.html"
+    Write-Host " "
+    Write-Host "IMPORTANT NOTE!!!"
+    Write-Host "The script aborted during the installation because you already have anaconda/miniconda installed. You are still missing important steps in the installation. Please uninstall Anaconda and run the script again"
+
```

Comment:
but the script does not stop here?
That might be hidden in the output from the user then?

Reply:
adressed in  https://github.com/dtudk/pythonsupport-scripts/pull/51/commits/e30956785a294e37245ab4a9167f72f39c23e811